app/services/utm_service.rb

```diff
@@ -20,14 +20,14 @@ class UTMService
     participant_validation_research: "participant-validation-research",
     participant_validation_sit_notification: "participant-validation-sit-notification",
     check_ect_and_mentor_info: "check-ect-and-mentor-info",
-    ects_to_add_validation_information: "ects-to-add-validation-information",
     fip_mentors_to_add_validation_information: "fip-mentors-to-add-validation-information",
     cip_mentors_to_add_validation_information: "cip-mentors-to-add-validation-information",
     induction_coordinators_who_are_mentors_to_add_validation_information: "induction-coordinators-who-are-mentors-to-add-validation-information",
     asked_ects_and_mentors_for_information: "asked-ects-and-mentors-for-information",
     sit_to_complete_steps: "sit-to-complete-steps",
     year2020_nqt_invite_school: "year2020-nqt-invite-school",
     year2020_nqt_invite_sit: "year2020-nqt-invite-sit",
+    "2109": "2109",
```

Comment:
uhh, what? What is 2109? Can we make it more human readable?

Reply:
Why does it need to be human readable? I can change it if you feel strongly but this was agreed with Shani.

Reply:
So the "why" is "for the sanity of whoever uses it" 😂 

This stuff is used in google analytics to monitor how the users got to our service. And if we start making it into magic numbers, someone will have to make a spreadsheet somewhere mapping those numbers to the meaning behind them.

Also, our sanity :D

Reply:
Also, what is 2109? 21st of September?

Reply:
If it is, maybe `2021-09-21` or something would convey that more obviously? For a moment there I thought it was a typo of 2019 😂

Reply:
Yes but there is also a balance to be struct with how long we end up making the url as apparently users are less likely to click on longer urls with very descriptive names. I think `ect-validation-info-2109` is sufficient enough to identify which email it is, and is also being logged in the spreadsheet